src/v2/Components/Artwork/GridItem.tsx

```diff
@@ -117,6 +117,7 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
         artwork={artwork}
         isHovered={isHovered}
         shouldShowHoverSaveButton={!!shouldShowHoverSaveButton}
+        contextModule={contextModule || ContextModule.artworkGrid}
```

Comment:
**thought(non-blocking):** what do you think of using `??` in favor of `||` here?

```suggestion
        contextModule={contextModule ?? ContextModule.artworkGrid}
```

since `??` will return the right value when `undefined` or `null` which will never happen while `||` will return the right value if left is falsy, but falsy in our case will be only `undefined`.

Does it makes sense? What do you think?